packages/graphback-datasync/tests/__snapshots__/DataSyncPluginTest.ts.snap

```diff
@@ -19,11 +19,9 @@ type Comment {
 
 type CommentDelta {
   id: ID!
-  title: String!
-  description: String!
-
-  \\"\\"\\"@index(name: 'Datasync_lastUpdatedAt')\\"\\"\\"
-  _lastUpdatedAt: GraphbackTimestamp
+  title: String
+  description: String
+  noteId: ID
```

Comment:
That is what was required. Question - would we be able to return that from the database?

Reply:
Return the column info?

Reply:
What this means

Reply:
> would we be able to return that from the database?

Can you explain what this means? I was asking if you want to return column information from database.

Reply:
I wasn't sure if knext will return it but I see that this works.

Reply:
http://knexjs.org/#Builder-columnInfo

Reply:
graphql-migrations uses it:L https://github.com/aerogear/graphback/blob/83a80cdbb1104da7b36acdfa54b37a871c3ff1a0/packages/graphql-migrations/src/connector/read.ts#L84